src/PlatformSockets.c

```diff
@@ -80,6 +95,9 @@ SOCKET bindUdpSocket(int addrfamily, int bufferSize) {
     int err;
 
     LC_ASSERT(addrfamily == AF_INET || addrfamily == AF_INET6);
+#ifdef __vita__
+    addrfamily = AF_INET;
```

Comment:
i guess vita system not support ipv6. 
so want to prevent ipv6 protocol. (actually it originally xyzz's works)
now i change to `LC_ASSERT` like this cases (but it can make side effect on release mode.)